.github/workflows/develop.yaml

```
@@ -55,6 +61,6 @@ jobs:
             push: true
             cache-from: type=gha
             cache-to: type=gha,mode=max
-            tags: netboxlabs/orb-agent:develop
+            tags: netboxlabs/orb-agent:${{ github.event.inputs.docker_tag }}
```

On `push` it won't default to `develop`, hence I recommend following change:


```suggestion
            tags: netboxlabs/orb-agent:${{ github.event.inputs.docker_tag || 'develop' }}
```

---

Thanks, good spot